docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+elasticsearch-173:
```

Comment:
Here I plan to make extendable docker-compose files available: https://docs.docker.com/compose/extends/

I already did it for manual testing environments: https://github.com/elastic/beats/tree/master/testing/environments